build.gradle.kts

```diff
@@ -108,10 +110,26 @@ tasks {
         archiveVersion.set("")
     }
 
-    test {
+    val junitTest by creating(Test::class) {
+        useJUnitPlatform {
+            includeEngines("junit-jupiter")
+        }
+        include("**/*Test.class")
+        testLogging.showStandardStreams = true
+    }
+
+    val spekTest by creating(Test::class) {
```

Comment:
Har laget to ulike måter å kjøre testene på. I IntelliJ opplevde jeg at når jeg prøvde å kjøre Junit-testen, så kjørte den i tillegg alle spek-testene. Dette er for å kunne kjøre de separat (ved klikk på kodelinja i IDEen). Når alt er over på Junit, kan jeg fjerne dette 👍🏼